CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# CONTRIBUTING
+
+## Markdown Guidlines
+
+Although we use the MDX format which sometimes allows the instantiation of React components in mdx files (see the [Readme section on this subject](/README.md#MDX)), it is recommended to avoid doing so to facilitate the contribution.
+
+Prefer Markdown over strings for translations, as soon as there are multiple paragraphs.
+
+## Local creation of services from contribution interface
+
+If you are interested in setting up a local instance where you can directly save the result of the contribution interface (`/contribute`) in the corresponding folder in Open Terms Archive project you can add the following constant to your `.env.local`
```

Comment:
If you are interested in setting up a local instance where you can directly save the result of the contribution interface (`/contribute`) in the corresponding folder in Open Terms Archive project you have to specify the path with the following constant in `.env.local` file: